src/test/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/quota/v5_4/OpenNebulaQuotaPluginTest.java

```diff
@@ -0,0 +1,234 @@
+package cloud.fogbow.ras.core.plugins.interoperability.opennebula.quota.v5_4;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.opennebula.client.Client;
+import org.opennebula.client.user.User;
+import org.opennebula.client.user.UserPool;
+import org.opennebula.client.vnet.VirtualNetworkPool;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+
+import cloud.fogbow.common.exceptions.FogbowException;
+import cloud.fogbow.ras.api.http.response.quotas.allocation.ResourceAllocation;
+import cloud.fogbow.ras.core.TestUtils;
+import cloud.fogbow.ras.core.datastore.DatabaseManager;
+import cloud.fogbow.ras.core.plugins.interoperability.opennebula.OpenNebulaBaseTests;
+import cloud.fogbow.ras.core.plugins.interoperability.opennebula.OpenNebulaClientUtil;
+
+@PrepareForTest({ OpenNebulaClientUtil.class, DatabaseManager.class })
+public class OpenNebulaQuotaPluginTest extends OpenNebulaBaseTests {
+
+    private static final String CPU_MAX_VALUE = "800";
+    private static final String CPU_USED_VALUE = "200";
+    private static final String DISK_MAX_VALUE = "9216";
+    private static final String DISK_USED_VALUE = "4096";
+    private static final String FRACTION_RESOURCE_USED_VALUE = "1.5";
+    private static final String MEMORY_MAX_VALUE = "15155";
+    private static final String MEMORY_USED_VALUE = "2048";
+    private static final String NETWORK_USED_VALUE = "0";
+    private static final String PUBLIC_IP_MAX_VALUE = "5";
+    private static final String PUBLIC_IP_USED_VALUE = "1";
+    private static final String VMS_MAX_VALUE = "8";
+    private static final String VMS_USED_VALUE = "1";
+    
+    private static final int FAKE_PUBLIC_NETWORK_ID = 100;
+    
+    private OpenNebulaQuotaPlugin plugin;
+    private User user;
+    
+    @Before
+    public void setUp() throws FogbowException {
+        super.setUp();
+        this.plugin = Mockito.spy(new OpenNebulaQuotaPlugin(this.openNebulaConfFilePath));
+        this.user = Mockito.mock(User.class);
+    }
+    
+    // test case: When calling the getUserQuota method, it must verify
+    // that the call was successful.
+    @Test
+    public void testGetUserQuota() throws FogbowException {
```

Comment:
Use the buildUsedAllocation and buildTotalAllocation methods instead of mocked allocations to increase test efficiency, also you may add a assertEquals to verify if the ResourceQuota object returned by the plugin's method is the same as a new one built with the total and used allocations generated previously